src/components/AboutPage.js

```diff
@@ -1,13 +1,88 @@
 import React from 'react';
+import '../styling/about.css';
+
+{/* TODO 
+- scrolling
+- request a project button
+- open projects button
+- style cleanup
+- spacing cleanup
+*/}
 
 const AboutPage = () => (
   <div class="container mt-5">
     <div class="row">
       <div class="col"></div>
       <div class="col-8">
-        <h1>About us</h1>
+        <h1>Build for Black Lives</h1>
+        <p class="p">
```

Comment:
if you get rid of class=p here, the css rule can just look like
```
p { ... }
```
instead of .p